apps/selvbetjening-ui/src/inntektsjustering/InntektsjusteringOppsummering.tsx

```diff
@@ -0,0 +1,88 @@
+import Navigation from "~components/Navigation";
+import React from "react";
+import {inntektsjusteringPath} from "~inntektsjustering/Inntektsjustering";
+import {IInntektsjusteringForm, IInntektsjusteringLagre} from "~inntektsjustering/InntektsjusteringDto";
+import {saveInntektsjustering} from "~api/api";
+import {useFormContext} from "react-hook-form";
+import {Alert, Heading} from "@navikt/ds-react";
+import styled from "styled-components";
+
+export default function InntektsjusteringOppsummering() {
+
+    const {
+        getValues,
+        formState: {errors},
+    } = useFormContext<IInntektsjusteringForm>()
+
+    const {
+        skalHaInntektNorge,
+        skalHaInntektUtland,
+        skalHaArbeidsinntekt,
+        skalHaNaeringsinntekt,
+        arbeidsinntekt,
+        naeringsinntekt,
+        arbeidsinntektUtland,
+        naeringsinntektUtland
+    } = getValues()
+
+    const onSubmit = async (inntektsjustering: IInntektsjusteringForm) => {
+        const {skalHaInntektNorge, skalHaInntektUtland, skalHaArbeidsinntekt, skalHaNaeringsinntekt} = inntektsjustering
+        const dto: IInntektsjusteringLagre = {
```

Comment:
Bare forkorte det tile `IIntekstjustering`? Da slipper vi enda et interface som bare skal konvertere dataen vår

Reply:
Altså slå sammen IInntektsjustering og IInntektsjusteringLagre? Og gjøre endre tidspunkt til string/undefined?
Har ikke så sterk formening om det frontend men backend er det ofte kjekt å ha en egen klasse for å lagre for å slippe at flere felter er optional frem til lagring men ikke trenger å være det etter.